checker/jtreg/README

```diff
@@ -8,3 +8,7 @@ There are a few cases when jtreg is appropriate:
    in the test file itself.
  * The jtreg approach makes it possible to check that an error message
    was output two times.
+
+Sometimes you should put tests in a directory such as
```

Comment:
The `nullness-extra` tests are not jtreg tests, so the reference is confusing.
Is this meant as an example for jtreg tests (they're not) or as an alternative to them?
A jtreg can also contain multiple files and even very flexibly indicate in which order they need to be compiled.
Can you clarify this paragraph?

Reply:
This is intended to be an alternative.  I updated the text; thanks for pointing out that it was unclear.

jtreg can handle some but not all cases with multiple files.  I tried to clarify that as well.